cg/services/order_validation_service/models/order.py

```diff
@@ -1,13 +1,13 @@
 from pydantic import BaseModel, Field
 
 from cg.constants import DataDelivery
-from cg.models.orders.orderform_schema import OrderCase
+from cg.constants.constants import Workflow
 
 TICKET_PATTERN = r"^#\d{4,}"
 
 
 class Order(BaseModel):
-    cases: list[OrderCase]
```

Comment:
All orders do not contain cases and as such, it does not make sense to keep cases in our base model for an order.